run.sh

```diff
@@ -2,7 +2,11 @@
 set -euxo pipefail
 if [ $GITHUB_EVENT_NAME = check_run ]
 then
-    RESULT=$(gh api /repos/$GITHUB_REPOSITORY/releases | jq -e -r '.[] | select(.draft == true and .name == "next") | .body' | egrep "$INTERESTING_CATEGORIES" || echo 'failed')
+    if [ -z "$RELEASE_DRAFT_BODY" ]; then
+        RELEASE_DRAFT_BODY="$(gh api /repos/$GITHUB_REPOSITORY/releases | jq -e -r '.[] | select(.draft == true and .name == "next") | .body')"
+    fi
+
+    RESULT=$(echo "$RELEASE_DRAFT_BODY" | egrep "$INTERESTING_CATEGORIES" || echo 'failed')
     if [[ $RESULT != 'failed' ]]; then
```

Comment:
BTW I think this could be simplified:
```suggestion
    if echo "$RELEASE_DRAFT_BODY" | egrep "$INTERESTING_CATEGORIES"
    then
```